src/pyggdrasil/tree_inference/_config.py

```diff
@@ -28,13 +34,38 @@ def id(self) -> str:
         return str_rep
 
 
+class MoveProbConfigOptions(Enum):
+    """Move probability configurations.
+
+    Implements configurations are DEFAULT and OPTIMAL from
+    SCITE paper, supplement p.15.
+
+    Default values:
+        prune_and_reattach=0.1,
+        swap_node_labels=0.65,
+        swap_subtrees=0.25
+
+    Optimal values:
+        prune_and_reattach=0.55,
+        swap_node_labels=0.4,
+        swap_subtrees=0.05
+
+        (`Optimal values find ML tree up to 2 or 3 times faster`)
+    """
+
+    DEFAULT = MoveProbConfig()
+    OPTIMAL = MoveProbConfig(
+        prune_and_reattach=0.55, swap_node_labels=0.4, swap_subtrees=0.05
+    )
+
+
 class McmcConfig(BaseModel):
     """Config for MCMC sampler."""
 
-    move_probs: MoveProbConfig
+    move_probs: MoveProbConfig = MoveProbConfigOptions.DEFAULT.value
     fpr: confloat(gt=0, lt=1) = 1.24e-06  # type: ignore
     fnr: confloat(gt=0, lt=1) = 0.097  # type: ignore
-    n_samples: conint(gt=-1)  # type: ignore
+    n_samples: conint(gt=-1) = 12000  # type: ignore
```

Comment:
What's the reason behind `-1`?

Reply:
The number of samples is supposed to be larger than -1, or actually larger than 0 I suppose.